readme.md

```diff
@@ -56,13 +56,19 @@ banner-HD.jpg 2x, banner-phone.jpg 100w, banner-super-HD.jpg 3x
 
 ## API
 
-### .parse()
+### .parse(string, [options])
 
 Accepts a srcset string and returns an array of objects with the possible properties: `url` (always), `width`, `density`.
 
-### .stringify()
+If options is set to `{strict: false}`, it will attempt to parse invalid input. Otherwise errors are thrown on invalid input.
 
-Accepts an array of objects with the possible properties: `url` (required), `width`, `density` and returns a srcset string.
+### .stringify(array, [options])
+
```

Comment:
Use this readme format for the readme API docs: https://github.com/sindresorhus/boxen#api